src/main/java/com/sequenceiq/cloudbreak/service/stack/resource/ResourceBuilder.java

```diff
@@ -24,6 +24,10 @@
 
     Boolean stop(SSCO ssco, Resource resource);
 
+    List<Resource> buildNames(P po, int index, List<Resource> resources);
```

Comment:
This method builds resources not names, it's a bit confusing that it is called `buildNames`